app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `Person` class should have `wife` and `husband` attributes to store references to the spouse. These attributes are necessary for the `create_person_list` function to work correctly when setting the spouse relationships.